core/org.wso2.carbon.user.core/src/main/java/org/wso2/carbon/user/core/tenant/JDBCTenantManager.java

```diff
@@ -1154,7 +1154,7 @@ private ResultSet getTenantQueryResultSet(Connection dbConnection, String sorted
             prepStmt = dbConnection.prepareStatement(sqlQuery);
             prepStmt.setInt(1, offset);
             prepStmt.setInt(2, limit);
-        } else if (dbType.contains("oracle")) {
+        } else if (dbType.contains("oracle") || dbType.contains("Oracle")) {
```

Comment:
Shall we change to equalIgnorecase and do it for other dbType as well?

Reply:
fixed